handlers/handlers.js

```diff
@@ -122,11 +122,20 @@ const deregisterHandler = async (client, msg) => {
   }
 };
 
+const chatHandler = (client, text) => {
+  fs.readFile(../chatbot/phraselist.txt, function(err, data){
```

Comment:
IF I EVER SEE ANOTHER ANONYMOUS FUNCTION THAT ISN'T AN ARROW FUNCTION YOU ARE BANNED

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions